RecoTracker/FinalTrackSelectors/python/preDuplicateMergingGeneralTracks_cfi.py

```diff
@@ -43,3 +43,30 @@
     copyExtras = True,
     makeReKeyedSeeds = cms.untracked.bool(False)
 ))
+
+# For Phase2PU140
+from RecoTracker.FinalTrackSelectors.earlyGeneralTracks_cfi import earlyGeneralTracks as _earlyGeneralTracks
+eras.trackingPhase2PU140.toReplaceWith(preDuplicateMergingGeneralTracks, _earlyGeneralTracks.clone(
```

Comment:
I'd suggest to clone from `_trackListMerger` (as for `trackingPhase1PU70` above) instead. I don't remember exactly why I did it that way for `trackingPhase1PU70`, probably for clarity (avoiding one level of cloning). You only need to add
```
    copyExtras = True,
    makeReKeyedSeeds = cms.untracked.bool(False)
```
at the end of the clone.